subiquity/models/filesystem.py

```diff
@@ -702,6 +702,16 @@ def boot(self):
         from subiquity.common.filesystem import boot
         return boot.is_bootloader_partition(self)
 
+    @property
+    def estimated_min_size(self):
+        fs_data = self._m._probe_data.get('filesystem', {}).get(self._path())
+        if fs_data is None:
+            return -1
+        val = fs_data.get('ESTIMATED_MIN_SIZE')
+        if val == 0:
+            return self.device.alignment_data().part_align
+        return val
```

Comment:
I don't know how likely this is to be a problem in practice but this should round the probert supplied value to part_align, and also round any value smaller than min_gap_size to min_gap_size (I'm now getting side-tracked by the design of PartitionAlignmentData -- how like is it that part_align and min_gap_size will be different? Why is it called min_gap_size and not min_part_size?)

Reply:
Er yes, I guess min(align_up(ESTIMATED_MIN_SIZE), actual_size) or something.